lib/smart_answer_flows/marriage-abroad/outcomes/outcome_opposite_sex_marriage_in_kosovo_when_residing_in_uk_or_kosovo.govspeak.erb

```diff
@@ -97,7 +98,8 @@
 
     You normally have to pay fees for consular services in the local currency - these are shown in the list of [consular fees for <%= calculator.country_name_lowercase_prefix %>](/government/publications/<%= calculator.ceremony_country %>-consular-fees).
```

Comment:
I am planning to address that text separately. I've updated the PR description to make that clearer.